### PR TITLE
AmiraTreeD updated

### PR DESCRIPTION
**Pull request by josmangarsal**
- Normalised space between 0 and 1
- Data optimun plane
- README.md updated with Amira setup and pictures
- Amira full example files

### CODE REVIEW

**Comment by mattmilten** [comment]
Nice! Can you please make the gif a bit smaller? Currently, it has almost 5 MB. You may want to check out this page https://ezgif.com/ , that's how I modified the plotly gif.

File: README.md [hunk 1]
```diff
@@ -14,7 +14,21 @@
...
+`AmiraTreeD.py` generates the '.am' data files to be loaded by Amira software to draw the tree using LineRaycast.
+
+### Settings
+
+![Project View](imgs/ProjectView.png)
+
+- `DataTree.am`: SpatialGraph data file with tree nodes and edges.
+- `LineRaycast`: Module to display the SpatialGraph. Note that is needed to set the colormap according to py code output (For instance 'Color map from 1 to 70' in this picture).
+- `DataOpt.am`: SpatialGraph data file with optimun value.
+- `Opt Plane`: Display the optimun value as a plane.
```
Comment (mattmilten) [inline README.md]: typo: "optimal"

**Comment by mattmilten** [comment]
oh, and just in general: git commit messages should always look like this:

```
header line (max 50 chars)
(empty line)
body (max 75 chars per line)
```

Otherwise, tools like GitHub might have difficulties displaying the commit properly (just look at your commits in this pull request :smiley: )

**Comment by josmangarsal** [comment]
ok! Thanks for the comments.
Changes:
- res folder.
- amira tree gif reduced, white background and no axes.

File: README.md (hunk 1 above)
Comment (josmangarsal) [reply README.md]: Done!

**Comment by mattmilten** [comment]
Very nice!

**Pull request merged**